layers/+emacs/org/packages.el

```diff
@@ -544,12 +544,33 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
 (defun org/init-org-brain ()
+  (use-package ascii-art-to-unicode
+    :defer t
+    :init
+    (add-hook 'org-brain-after-visualize-hook #'aa2u-buffer))
   (use-package org-brain
     :defer t
     :init
     (progn
+
+      (spacemacs/declare-prefix "aoB" "org-brain")
       (spacemacs/set-leader-keys
-        "aob" 'org-brain-visualize)
+        "aoBv" 'org-brain-visualize
+        "aoBa" 'org-brain-agenda)
+
+      (spacemacs/declare-prefix-for-mode 'org-mode "mB" "org-brain")
+      (spacemacs/set-leader-keys-for-major-mode 'org-mode
+
```

Comment:
Could you also add a binding for `org-brain-visualize` for the `major-mode`?
I have also noticed some missing prefix declarations for the new bindings.

Should be something of the line:
```suggestion
      (spacemacs/declare-prefix-for-mode 'org-mode "mB" "org-brain")
      (spacemacs/declare-prefix-for-mode 'org-mode "mBa" "add")
      (spacemacs/declare-prefix-for-mode 'org-mode "mBg" "goto")
      (spacemacs/set-leader-keys-for-major-mode 'org-mode

        "Bac" 'org-brain-add-child
        "Bap" 'org-brain-add-parent
        "Baf" 'org-brain-add-fiendship

        "Bgc" 'org-brain-goto-child
        "Bgp" 'org-brain-goto-parent
        "Bgf" 'org-brain-goto-friend

        "BR"  'org-brain-refile
        "Bx"  'org-brain-delete-entry)
```